ios/FluentUI/Resources/Localization/LocalizedStrings.swift

```diff
@@ -0,0 +1,15 @@
+//
```

Comment:
Probably don't want a generic API for exposing localized strings per offline conversation so let's not add this file. Instead just expose this specific string from NavigationBar.swift?

Reply:
Done